# generated by patch-package 0.0.0
#
# declared package:
#   @types/a: file:@types+a-1.0.0.tgz
#
diff --git a/node_modules/@types/a/index.d.ts b/node_modules/@types/a/index.d.ts
index 9556c8b..ef170e3 100644
--- a/node_modules/@types/a/index.d.ts
+++ b/node_modules/@types/a/index.d.ts
@@ -1,4 +1,4 @@
-export default interface a_type {
+export default interface patch_package {
   a: number,
   b: number,
   c: number,
